src/SAML2/XML/samlp/AttributeQuery.php

```diff
@@ -116,28 +93,53 @@ public function setAttributes(array $attributes): void
 
```

Comment:
Here I think we should have a check for the type of the elements in `$attributes`.